packages/components/src/docs/General/10-FAQ.mdx

```diff
@@ -0,0 +1,71 @@
+
+### How should I be using the Solid Design System and why?
+Considering the query above, our components, styles, samples, and patterns are designed to be applicable across various web contexts, ensuring the creation of fully accessible, design standards-compliant, and lightweight websites or applications.
+
+### What are the 3 core principles of the Solid Design System?
+Solid Components, as part of the Solid Design System, serving as a robust foundation for a digital presence, built upon three core principles:
+1. 🌟 Usable & consistent
+2. 🧩 Reusable & flexible
+3. 🛠️ Maintainable & reliable
+Visit our [introduction page](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-introduction--docs#principles) for more details.
+
+### How do components, styles, samples, and patterns differ in our library?
+In our library, we offer four distinct types of elements:
+1. Components: These are lightweight [web components](https://developer.mozilla.org/en-US/docs/Web/API/Web_Components) constructed with lit-element.
+2. Styles: These are even lighter CSS classes that can be applied to any HTML element.
+3. Samples: These are practical demonstrations showcasing the usage of our components and styles within specific contexts.
+4. Patterns: These are predefined use-cases designed to address specific design requirements without the need for a dedicated component.
+Visit our [introduction page](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-introduction--docs#structure) for more details.
+
+### Which browsers are supported?
+TBA
+
+### How can I submit a feature request or report a bug?
+To do so, navigate to the issue section on GitHub and utilize our provided issue templates. Whether you're suggesting a new feature or reporting a bug, these templates offer placeholders and designated content sections where you can input the necessary information. Please ensure to include as many detailed specifics as possible when submitting your request or bug report.
+
+### Is there an endpoint to the development of the Solid Design System?
+No, the Solid Design System is an ongoing endeavor, much like the ever-evolving nature of the web itself. We are committed to continually refining and expanding its features, ensuring it remains aligned with the latest standards and technologies.
+
+### Is there a roadmap for upcoming features?
+Sure. You can explore our project board for a detailed overview of our [roadmap](https://github.com/orgs/solid-design-system/projects/1/views/48).
+
+### What we do and what we don't do?
+- ✅ Provide guidance and support with components, styles, samples and patterns
+- ✅ Create, update and maintain our components, styles, samples and patterns
+- ✅ Write documentation about the system’s elements’ behaviours and interactions
+- tba
+
+- ❌ Screen designs
```

Comment:
maybe more "Complete screen designs"

Reply:
Gain stakeholder and consumer feedback and requests to base our work on insights and needs.
Fosters contribution and engagement from product teams.
Service and support: office hours, trainings, consulting, design system community
Advocating for adoption the solid design system.
Onboarding new product teams.

Reply:
@coraliefeil I added your points.

@mariohamann I am irritated here. We do NOT offer complete screen designs but components to create them. Or was your point mor on the "what we dont do"?

Reply:
@mariohamann forget it... VSC is showing me the comments in the wrong line... :(

Reply:
Yeah, so we create "Screen designs" – but not like fully fledged apps.